Generator/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM circleci/python:3.7.0
+
+ADD . /
+RUN sleep 60
+
+CMD bash generator.sh
```

Comment:
Please, add a newline here and consider https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file